src/services/home-dashboard/modules/PersonalHealthDashboard.vue

```diff
@@ -304,4 +323,8 @@ export default {
         }
     }
 }
+:deep(.p-empty) {
+    padding-top: 2.5rem;
+    padding-bottom: 1.56rem;
```

Comment:
nitpick: If possible, set the rem unit to 2px :) In this case, padding-bottom should be `1.5rem`
![스크린샷 2023-03-03 오전 10 31 57](https://user-images.githubusercontent.com/18563857/222608927-ad694ce8-ddf6-4e18-92b9-91ec201f2f2e.png)